automation/README.md

```diff
@@ -0,0 +1,77 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
+which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+
+This script will execute the Ansible playbook [`./control.yml`](../control.yml)
+which wraps entire testing flow.
+
+
+## `control.yml` Ansible playbook
+
+This Ansible playbook is wrapper for entire testing flow which is composed from
```

Comment:
s/wrapper/a wrapper/   
s/entire/the entire/

Reply:
Addressed